controllers/endpoints-controller.js

```diff
@@ -0,0 +1,109 @@
+exports.getEndpoints = (req, res, next) => {
```

Comment:
Similarly in here Eloise - you'll need to keep updating this controller. You can use the regular JS `require` to bring in a JSON file 👍